kubernetes/src/main/java/io/kubernetes/client/ApiClient.java

```diff
@@ -768,10 +768,10 @@ public String selectHeaderAccept(String[] accepts) {
      *
      * @param contentTypes The Content-Type array to select from
      * @return The Content-Type header to use. If the given array is empty,
-     *   JSON will be used.
+     *    or matches any, JSON will be used.
      */
     public String selectHeaderContentType(String[] contentTypes) {
-        if (contentTypes.length == 0) {
```

Comment:
> ... and I don't think we can land it there.
We did this for python client. https://github.com/swagger-api/swagger-codegen/pull/4152
I think it is a reasonable thing to do because swagger clients expect json and when server accept *, it includes json.
> The right thing to do is to fix the Kubernetes swagger to not have this.
I also agree with this regardless of swagger-codegen fix, we need to fix this in our spec.